### DISCUSSION TITLE
GitHub is not accepting our updated branches and allowing us to merge new commits.

### DISCUSSION THREAD

**Discussion by samsquamch53** [community/community#51771] [New to GitHub]
I think your product is more trouble than it is worth.  I am taking an intro to software engineering class and the teacher is making us use GitHub because it is "user friendly" and intuitive.  I sat through your tutorial exercise and it did not mention how to incorporate code worked on in visual studios into your GitHub repository.  This product is terrible for beginners because you don't provide adequate introductory training documentation or helpful tutorials. I will be referring anyone that asks me about GitHub to not use it.

**Comment by queenofcorgis** [comment]
Hey there @samsquamch53 welcome to the world of GitHub, we're glad you've joined us. First, thank you for the candid feedback - we really appreciate it. I wanted to share some additional resources to help get you started:

- This [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) includes links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.
- [GitHub Skills](https://skills.github.com/) for some beginner tutorials
- [Intro to Using GitHub with VSCode](https://www.youtube.com/watch?v=i_23KUAEtUM) and some more documentation [here](https://code.visualstudio.com/docs/sourcecontrol/intro-to-git). 

As you continue on your journey - if there are areas you feel need improvement - feel free to share them here on this thread so we can continue to improve.
